packages/react/src/isomorphicClerk.ts

```diff
@@ -314,6 +320,15 @@ export default class IsomorphicClerk {
     }
   }
 
+  get telemetry(): TelemetryCollector | undefined {
+    if (this.clerkjs) {
+      // @ts-expect-error -- We can't add the type here due to the TelemetryCollector type existing in shared, but the Clerk type existing in types
+      return this.clerkjs.telemetry;
+    } else {
+      return undefined;
+    }
+  }
```

Comment:
This will allow us to record telemetry via the `useClerk` hook, which will let us instrument more easily in our components.